README.md

```diff
@@ -16,115 +16,146 @@
+           ('The Grand Cultural Influence of Octavia Butler',
+            3, '2019-06-21', 3.80, '<p>Tomorrow, June 22, would have been legendary SF novelist and short story writer Octavia Butler’s 72nd birthday. She died in 2006—much too young, at only 58—already a certified genius who had a profound impact on many readers and writers across the world. Not surprisingly, this includes many of the best writers of SF, fantasy, speculative fiction, and horror working today, and so to celebrate Butler’s birthday, I’ve collected a few of their thoughts on her influence.</p>'),
+           ('What Gets Lost (and Found) in Translating Prose to Comics',
+            8, '2019-01-08', 4.60, '<p>As goes language, so too goes form. Comics creators have long drawn on literary sources for inspiration or outright adaptation. We’ve moved far beyond the days of <em>Classics Illustrated</em>, wherein a condensed version of a novel was translated in a straightforward manner to words and pictures on a page. David Mazzucchelli’s adaptation of Paul Auster’s <em>City of Glass</em> is a notable work in its own right, and one that neatly blends the aesthetics of two distinctive artists.</p>'),
+           ('Why We’ll Never Get Tired of Literary Retellings',
+            5, '2019-05-28', 4.95, '<p>I make an effort to read a wide variety of books, but the genre that I find myself drawn to time and again is retellings of classic novels. I’ve yet to hear about a new twist on a timeless tale that doesn’t sound compelling to me. In fact, it’s a niche of storytelling that I enjoy so much that I’ve written my own entry into the genre, a novel entitled <em>You, Me, and the Sea</em> that is inspired by Emily Bronte’s <em>Wuthering Heights</em>. As I worked on my novel, I spent a lot of time thinking about my favorite retellings, and what it is, exactly, that makes these stories so endearing to readers—and writers. What I’ve come to believe is that reimaginings present a unique delight to readers because they manage to combine the pleasure of surprising twists with the comfort of a familiar story.</p>')
 
 5. Retrieve articles with the information about the author attached to each row (there should be 12 rows in the result and around 10 columns, including the article’s title, text, rating, and date as well as the author’s name and sex):
 
-    ```postgresql
-    ... here goes your SQL ...
-    ```
+    SELECT * FROM authors FULL JOIN articles ON authors.id = aricles.author;
```

Comment:
`...should be 12 rows in the result...` you use `FULL JOIN` for it.
in the next task, it returns `FULL JOIN: ? rows - 13`
how it can be?

Reply:
you need to select from articles
`Retrieve articles with the information...`